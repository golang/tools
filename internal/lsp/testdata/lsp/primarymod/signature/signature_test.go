package signature_test

import (
	"testing"

	"golang.org/x/tools/internal/lsp/signature"
)

func TestSignature(t *testing.T) {
	signature.AliasSlice()    //@signature(")", "AliasSlice(a []*signature.Alias) (b signature.Alias)", 0)
	signature.AliasMap()      //@signature(")", "AliasMap(a map[*signature.Alias]signature.StringAlias) (b map[*signature.Alias]signature.StringAlias, c map[*signature.Alias]signature.StringAlias)", 0)
	signature.OtherAliasMap() //@signature(")", "OtherAliasMap(a map[signature.Alias]signature.OtherAlias, b map[signature.Alias]signature.OtherAlias) map[signature.Alias]signature.OtherAlias", 0)
}
