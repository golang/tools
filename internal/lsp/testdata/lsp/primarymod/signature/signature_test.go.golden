-- AliasMap(a map[*signature.Alias]signature.StringAlias) (b map[*signature.Alias]signature.StringAlias, c map[*signature.Alias]signature.StringAlias)-signature --
AliasMap(a map[*signature.Alias]signature.StringAlias) (b map[*signature.Alias]signature.StringAlias, c map[*signature.Alias]signature.StringAlias)

-- AliasSlice(a []*signature.Alias) (b signature.Alias)-signature --
AliasSlice(a []*signature.Alias) (b signature.Alias)

-- GetAlias() signature.Alias-signature --
GetAlias() signature.Alias

-- GetAliasPtr() *signature.Alias-signature --
GetAliasPtr() *signature.Alias

-- OtherAliasMap(a map[signature.Alias]signature.OtherAlias, b map[signature.Alias]signature.OtherAlias) map[signature.Alias]signature.OtherAlias-signature --
OtherAliasMap(a map[signature.Alias]signature.OtherAlias, b map[signature.Alias]signature.OtherAlias) map[signature.Alias]signature.OtherAlias

-- SetAliasSlice(a []*signature.Alias)-signature --
SetAliasSlice(a []*signature.Alias)

-- SetOtherAliasMap(a map[*signature.Alias]signature.OtherAlias)-signature --
SetOtherAliasMap(a map[*signature.Alias]signature.OtherAlias)

